WordPress/Classes/ViewRelated/Site Creation/Final Assembly/SiteAssemblyContentView.swift

```diff
@@ -104,12 +140,16 @@ final class SiteAssemblyContentView: UIView {
     }
 
     let siteCreator: SiteCreator
+    private lazy var shouldShowDomainPurchase: Bool = {
```

Comment:
nit: perhaps we should add a `\n` between `siteCreator` property and `shouldShowDomainPurchase`.